packages/web-app-files/src/components/FilesList/KeyboardActions.vue

```diff
@@ -86,6 +86,8 @@ export default defineComponent({
     }),
 
     areCustomKeyBindingsEnabled() {
+      const activeElementTag = document.activeElement.tagName
+      if (['textarea', 'input', 'select'].includes(activeElementTag.toLowerCase())) return true
```

Comment:
Personally not a fan of the inline return without  {}

Reply:
Same. Code style in web and related repos is to not inline return statements.

Reply:
Tried to find a eslint rule set plugin, but nothing found unfortantly